src/nvim/event/socket.c

```diff
@@ -21,58 +21,34 @@
 # include "event/socket.c.generated.h"
 #endif
 
-#define NVIM_DEFAULT_TCP_PORT 7450
-
-void socket_watcher_init(Loop *loop, SocketWatcher *watcher,
-    const char *endpoint, void *data)
+void socket_watcher_init(Loop *loop, SocketWatcher *watcher, const char *endpoint,
+                         void *data)
```

Comment:
Can we just remove `data` from the arguments? It's only called with `NULL` in `server_start()`. Maybe the original author planned ahead when he wrote that.